templates/terranix/flake.nix

```diff
@@ -0,0 +1,29 @@
+{
+  inputs = {
+    flake-utils.url = "github:numtide/flake-utils";
```

Comment:
@oscar-izval Since #40 was recently merged, I believe we should update all existing templates to use `flake-parts` instead, for consistency. What are your thoughts?

Reply:
Thought about that, but figured that the idea of a template is to be a relatively simple starting point. Besides, I don't think the internal implementation of the tool is relevant to the final user, having flake parts could make it harder for newcomers. Anything else I might have missed in this reasoning?

Reply:
@oscar-izval fair point; however, I'm wondering whether we should give a completely unopinionated template in that scenario, similar to what devenv provides with the [simple](https://github.com/cachix/devenv/blob/main/templates/simple/flake.nix#L15) template.

Reply:
In and ideal world, we'd ship templates with the most widely adopted patterns, but that's not an easy metric to find. Defaulting to an unopinionated template sounds good to be, specially if that's what devenv is doing.